policyengine_us/variables/gov/states/oh/tax/income/credits/oh_retirement_income_credit.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class oh_retirement_income_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"
+    defined_for = StateCode.OH
```

Comment:
Then remove from the formula
```suggestion
    defined_for = "oh_retirement_income_credit_eligible"
```